org_fedora_hello_world/dbus_service/installation.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+"""
+This module contains tasks of the D-Bus service.
+
+A task - Task subclass - performs actual work such as manipulating files.
+Tasks run in threads so as not to block communication with the service
+process. All installation jobs must be represented as Tasks.
```

Comment:
I think this should be:

"All installation **and configuration** jobs must be represented as Tasks."

Paging @poncovka for confirmation though. :)

Reply:
Fixed. I think that's what we want to say regardless of whether it's the whole truth now.